File: leo2-central/src/main/java/org/deku/leo2/central/data/repositories/SectorRepository.java
package org.deku.leo2.central.data.repositories;

import org.deku.leo2.central.data.entities.Sector;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QueryDslPredicateExecutor;
import org.springframework.stereotype.Repository;

/**
 * Created by masc on 16.05.15.
 */
public interface SectorRepository extends JpaRepository<Sector, Integer>, QueryDslPredicateExecutor {
}

File: leoz-node/src/main/java/org/deku/leoz/node/data/repository/master/SectorRepository.kt
package org.deku.leoz.node.data.repository.master

import org.deku.leoz.node.data.jpa.MstSector
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.querydsl.QuerydslPredicateExecutor

/**
 * Sector repository
 * Created by masc on 16.05.15.
 */
interface SectorRepository :
        JpaRepository<MstSector, Long>,
        QuerydslPredicateExecutor<MstSector>
